hs-abci-sdk/src/Tendermint/SDK/Application/Handlers.hs

```diff
@@ -165,18 +171,44 @@ makeHandlers (HandlersContext{..} :: HandlersContext alg ms core) =
         return . ResponseCommit $ def
           & Resp._commitData .~ Base64.fromBytes rootHash
 
+      beginBlock :: Handler 'MTBeginBlock (BA.BaseAppEffs core)
+      beginBlock (RequestBeginBlock bb) = do
+        catch
+          (do
+            res <- evalBeginBlockHandler $ M.applicationBeginBlocker app bb
+            return . ResponseBeginBlock $ res
+          )
+          (\(_ :: BA.AppError) -> return . ResponseBeginBlock $ def
+          )
+
+      endBlock :: Handler 'MTEndBlock (BA.BaseAppEffs core)
+      endBlock (RequestEndBlock eb) = do
+        catch
+          (do
+            res <- evalEndBlockHandler $ M.applicationEndBlocker app eb
+            return . ResponseEndBlock $ res
+          )
+          (\(_ :: BA.AppError) -> return . ResponseEndBlock $ def
```

Comment:
similar comment as above about the `Exception` message

Reply:
:+1: